front-end/src/styles/GlobalStyle.tsx

```diff
@@ -12,6 +12,18 @@ const GlobalStyle = createGlobalStyle`
     font-family: "Noto Sans KR";
     font-weight: 400;
   }
+
+  html,
+  body {
+    height: 100%;
+    width: 100%;
+    margin: 0;
+  }
+
+  #root {
+    height: 100%;
```

Comment:
height: 100% 를 적용하였을 때, 스크린의 높이만큼이 적용되게 하기 위한 css 설정입니다

Reply:
👍👍👍👍

Reply:
```suggestion

  html,
  body {
    margin: 0;
  }

  #root {
   width: 100vw;
   height: 100vh;
```
`스크린`이 기준이 될 것 같은데 위처럼 변경하면 다르게 동작할까요?

Reply:
viewport 는 스크롤 사이즈까지 포함된 개념이므로 width: 100vw, height: 100vh 로 주게되면 

1. width/height가 스크롤 사이즈가 포함된 전체 뷰포트 사이즈로 설정됨
2. 추가적으로 브라우저에 스크롤이 표시됨

=> 스크롤 사이즈만큼 추가로 width/height가 설정되게 되어, 의도와 다르게 렌더링됩니다.


스크롤 사이즈까지 고려하여 깔끔하게 스크린 사이즈를 맞추기 위해  html에 width,height 100%를 주는 것이 
일종의 CSS Trick인 것으로 이해하였습니다 :smile: 

```css
  html,
  body {
    height: 100%;
    width: 100%;
    margin: 0;
  }

```


<제안>
![image](https://user-images.githubusercontent.com/31029000/125477167-ad41cb10-f6fb-42f7-9398-dd8f84576f5f.png)

<기존>
![image](https://user-images.githubusercontent.com/31029000/125479224-fa72e0ab-a05a-47f2-9d3c-1503578e62f2.png)

Reply:
`vw/vh`는 스크린 포함이었군요 :) 
스크롤이 없는 `100%`가 훨씬 깔끔하네요 👍👍👍